charts/portworx/templates/storage-cluster.yaml

```diff
@@ -304,9 +304,16 @@ spec:
     {{- end }}
   {{- end }}
 
-  {{- if or (eq .Values.csi true) (eq .Values.csi false) }}
+  {{- if or (eq .Values.csi.enabled true) (eq .Values.csi.enabled false) }}
```

Comment:
So helm upgrade from current versions will not honour these previous value?

Reply:
The previous values will no longer be honored. To enable CSI, users must now set `Values.csi.enabled` to `true`.